javascript-light-bulb/main.js

```diff
@@ -5,7 +5,8 @@ var $backGround = document.querySelector('.background');
 
 function lightBulb(event) {
   clicked++;
-  if (clicked % 2 === 0) {
+  var lightOff = (clicked % 2 === 0);
```

Comment:
This is still using a variable holding the click count. Don't try to address feedback **_fast_**, try to address it **_well_**.